### ISSUE TITLE
use requests

### ISSUE THREAD

**Issue by Mouad-scriptz** [cubicbyte/reddit-account-generator#9]
use requests

**Comment by cubicbyte** [comment]
I'm a little confused about your issue

If you propose to use pure requests instead of selenium, it is impossible because of reddit protection.

**Comment by Mouad-scriptz** [comment]
I know this might sound like a lot, but Reddit's security measures are quite weak. They only check headers, and if you're using Chrome with TLS, they also inspect cookies and a hidden csrf_token on the page's html (/register). The most challenging part is solving the reCAPTCHA. nothing other then that

**Comment by Mouad-scriptz** [comment]
I would say just give it a try.

**Comment by Mouad-scriptz** [comment]
Also im not hating or anything your project is good but using requests would make it much faster and efficient.

**Comment by cubicbyte** [comment]
> The most challenging part is solving the reCAPTCHA. nothing other then that

That's exactly what I'm talking about

First of all, in order to invoke captcha at all, you can't just make a request to some URL. That query is generated by a ton of JS loaded by google, which needs to be interpreted, not parsed.

Second, the same applies to the captcha page itself. At least I haven't found any opensource solution for solving captchas this way.

So even if it is possible to do it, I won't be able to do it, I've already tried in other projects

**Comment by ZakariaMQ** [comment]
yeah, I did it without skipping the captcha. It got solved using the 2Captcha service
I used the post request to sign up and verify email all using httpx 
but the accounts got banned in a matter of hours

+ I did this on the new Reddit 

now I just need to figure out why accounts get suspended even if all headers are correct and so on

**Comment by Mouad-scriptz** [comment]
> yeah, I did it without skipping the captcha. It got solved using the 2Captcha service I used the post request to sign up and verify email all using httpx but the accounts got banned in a matter of hours
> 
> * I did this on the new Reddit
> 
> now I just need to figure out why accounts get suspended even if all headers are correct and so on

httpx is detected

**Comment by ZakariaMQ** [comment]
I tried both httpx and requests but same results

**Comment by ZakariaMQ** [comment]
this is just a captcha solver which I made in 20 lines of code the important thing is to make accounts live and not get banned after a few hours

**Comment by Ousret** [comment]
How about trying [Niquests](https://github.com/jawah/Niquests) instead?
It leverages a true HTTP/2 and HTTP/3 connection with multiplexing. If speed matters.
